LondonBritishLibrary/orient/BLorient3716a.xml

```diff
@@ -0,0 +1,134 @@
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d2">
+                                <desc>Coloured magical picture</desc>
+                            </decoNote>
+                            <decoNote type="miniature" xml:id="d3">
+                                <desc>Coloured magical picture</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Other">Preserved together with two other manuscripts in a box which also contains, 
+                                    suspended from a leather strap, seven <material key="leather"/>leather cases (4, cylindrical, for scrolls and 3 for 
+                                    amulet booklets), glass beads, pebbles, shells, and pieces of ivory. This box also contains an English translation of an 
+                                    amulet for protecting a mule.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1800" notAfter="1889">19th century</origDate></origin>
+                        <provenance>The name of the owner was 
+                            <persName role="owner" ref="PRS14432WalGabrAdallahu">Walatta Gabrǝʾel ʾAdallāhu</persName>.
+                            Presented by the <persName role="bequeather" ref="PRS14375Clarke"><roleName type="title">Revd.</roleName>
+                            Leop. Stanley Clarke</persName> <date when="1889-06-19">19 June 1889</date>.</provenance>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">76</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
```

Comment:
I would add "poetry" keyword.

Reply:
There is already "Poetry" in the list. Do you mean "Chants"?

```
<keywords>
                    <term key="ChristianLiterature"/>
                    <term key="Poetry"/>
                </keywords>
```